Week-12/todo-list/src/Components/Todo.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import '../Style/Todo.css';
+import ListItem from "./ListItem";
+
+const Todo = () => {
+
+    const [list, setList] = useState(['Coding', 'Eating', 'Sleeping', 'Debugging']);
+    const addItem = () => {
+        setList([...list, '']);
+    }
+    const removeItem = (index) => {
+        let newList = list;
+        newList.splice(index, 1);
+        setList([...newList]);
+    }
+    const updateItem = (index, value) => {
+        console.log(`Debounced list updated call with value: ${value}`);
+        let updatedList = list;
+        updatedList[index] = value;
+        setList([...updatedList]);
+    }
+    return (
+        <>
+            <div className="center">
+                <h1>TODO Application</h1>
+            </div>
+            <div className="rowCenter">
+                {
+                    list.map((itemName, i) => {
+                        return(
+                            <div className="smallHeight" key={`${itemName}-${i}`}>
+                                <ListItem name={itemName} index={i} removeItem={removeItem} updateItem={updateItem}/>
+                            </div>
+                        )
+                    })
+                }
+                <button className="AddButton" onClick={addItem}>Add</button>
```

Comment:
inappropriate class name, keep consistency in class naming.

Reply:
changed it to `addButton`, following camel casing throughout this assignment